### DISCUSSION TITLE
posts/2023/09/routine/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#27] [General]
# posts/2023/09/routine/

想不出标题，只能以日期暂代。

https://guozheng.rbind.io/posts/2023/09/routine/

**Comment by residualsun1** [comment]
评论试验。

**Reply by CyrusYip** [reply]
![image](https://github.com/residualsun1/Residualsun/assets/60951091/d90068a0-558f-446e-88b0-dfdd90faeab9)

这里末尾缺了个 `/`，修改后应该评论就和文章关联了。

**Reply by residualsun1** [reply]
试了一下好像不行，原评论应该是只能保留在原来的名称中。
